### DISCUSSION TITLE
Milvus 11.0/10.5 and GolangSDK

### DISCUSSION THREAD

**Discussion by Noomegipy** [milvus-io/milvus#4661] [Q&A and General discussion]
Hello!
Could you tell me please  what happened with 11.0 milvus? Its weird by it looks like deprecated, but 10.5 looks like actual. Which version should i take? I also ask because  golang sdk 0.45 using float32 in Entity struct -> 
it probably means less accurate search and i need float64 -  but it but it is only in the sdk version 5.0.0 for  milvus version 11.0.
Please help:)

Sorry if this is a frequent question

**Comment by wxyucs** [comment]
@Noomegipy Hi, version 0.11 is no longer actively maintained, please change to using version 0.10 or waiting for our next version. And vector is calculated using float32 in Milvus so that both v0.4.5 and v0.5.0 of the Go SDK vector passed as float32. :)

**Comment by gujun720** [comment]
Milvus 0.10.5 is the recommended version. You can find What happened for 0.11  in 
https://github.com/milvus-io/milvus/discussions/4640

**Comment by Noomegipy** [comment]
Thanks for the detailed answer!
